pkg/env/node_scan.go

```diff
@@ -32,4 +32,7 @@ var (
 
 	// NodeInventoryContainerEnabled is used to tell compliance whether a connection to the node-inventory container should be attempted
 	NodeInventoryContainerEnabled = RegisterBooleanSetting("ROX_CALL_NODE_INVENTORY_ENABLED", true)
+
```

Comment:
@Maddosaurus could I remove the following settings from this file (see lines above this one):

- NodeScanningCacheDuration 
- NodeScanningInitialBackoff 
- NodeScanningMaxBackoff 

They are unused in stackrox currently, cause we moved them to scanner. WDYT?

Reply:
Great Catch! 
These can be removed, yes.